Examples/calculator.lisp

```diff
@@ -30,17 +30,28 @@
   (make-text-style :sans-serif :roman :large))
 
 (defun calculator ()
-  (let ((frame (make-application-frame 'calculator)))
+  (let ((frame (make-application-frame 'calculator-app)))
     (run-frame-top-level frame)
     frame))
 
 (defun show (number)
   (setf (gadget-value (slot-value *application-frame* 'text-field))
 	(princ-to-string number)))
 
+(defvar *last-button* nil)
+
+(defmethod clim-internals::effective-gadget-background :around ((b push-button-pane))
+  (if (eq b *last-button*)
+      (clim-internals::make-rgb-color 0.9 0.2 0.2) 
+      (call-next-method)))
+
 (defun queue-number (number)
```

Comment:
is there a need for that in calculator example? I'm not especially keen with advertising internal interface.